### DISCUSSION TITLE
Qiitaの無断コピーサイト？

### DISCUSSION THREAD

**Discussion by JunichiIto** [increments/qiita-discussions#203] [ヘルプ] [answered]
Qiitaの記事を多数無断コピーしているサイトがありました。

例
https://jpdebug.com/p/2864286
<img width="1333" alt="Screen Shot 2022-04-12 at 10 22 11" src="https://user-images.githubusercontent.com/1148320/162859774-e72f02b8-9749-4dec-9f86-d56f415a94a9.png">

いちおうご丁寧に元記事のリンクは載せているのですが、「CC BY SA 2.5、CC BY SA 3.0とCC BY SA 4.0の下で認可されます」ってそんな話どこかにありましたっけ！？

<img width="938" alt="Screen Shot 2022-04-12 at 10 24 25" src="https://user-images.githubusercontent.com/1148320/162860081-d4c1eff0-629d-43f4-a89a-fa3d401f6c28.png">

サイトの運営者に連絡を取るなどして、無断コピーをやめさせた方がよいかと思います。
ご確認よろしくお願いします🙏

**Comment by ohakutsu** [comment, marked as answer]
@JunichiIto 
ご報告いただきありがとうございます！

今回ご報告いただいたサイトには、Qiita運営から警告を送っておきました :+1:
また、記事の著作権については[利用規約](https://qiita.com/terms)にある通り投稿者のみなさんに帰属しているので、ご安心ください。

今後、このようなコピーサイトや転載が起こった際、報告や申請をしやすくなるようにヘルプを整備しようと思います！

**Reply by JunichiIto** [reply]
ご対応どうもありがとうございます！今後の対応もよろしくお願いします〜。
